lib/Makefile

```diff
@@ -1,4 +1,4 @@
-phpspy_cflags:=-std=gnu2x -Wall -Werror -Wextra -pedantic -O3 $(CFLAGS)
+phpspy_cflags:=-std=gnu2x -Wall -Werror -Wextra -pedantic -O3 -Wno-unused $(CFLAGS)
```

Comment:
Is `-std=gnu2x` still needed with the removal of `maybe_unused` attributes? If it's not, it may be good to lower the requirements to support older compilers (the same applies to the same line).

Reply:
Good eye ;) Let me check.

Reply:
That would be working. Committed the change.